bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusHF/HPlusBottom_4FS_MuRFScaleDynX0p50_M125_TuneCP5_13TeV_amcatnlo_pythia8/HPlusBottom_4FS_MuRFScaleDynX0p50_M125_TuneCP5_13TeV_amcatnlo_pythia8_extramodels.dat

```diff
@@ -0,0 +1 @@
+loop_sm_MSbar_yb_yc.tar.gz
```

Comment:
this model is not in the central pool yet.

Reply:
Yes, this was already pointed out in the pull request comment (there is also a link to the model itself), how should we proceed to have it uploaded in the central pool?

Reply:
ok, it's in the central pool now

Reply:
Thank you!